<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KorttiWindow</class>
 <widget class="QDialog" name="KorttiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankSimul</string>
  </property>
  <widget class="QLabel" name="labelidkortti">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>741</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="stackMenu">
    <widget class="QPushButton" name="btnNostaRahaa">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nosta rahaa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaldo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Saldo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSiirraRahaa">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>290</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Siirr√§ rahaa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTilitapahtumat">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tilitapahtumat</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stackSaldo"/>
   <widget class="QWidget" name="stackTapahtuma">
    <widget class="QTextEdit" name="textTilitapahtumat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>561</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_tilitapahtumat">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>651</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_vanhemmat">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>390</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_uudemmat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stackNosto"/>
   <widget class="QWidget" name="stackSiirto"/>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Palaa valikkoon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>labelidkortti</zorder>
  <zorder>btnReturn</zorder>
  <zorder>btnLogout</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
